packages/cadl-ranch-specs/http/azure/core/basic/mockapi.ts

```diff
@@ -77,7 +77,7 @@ Scenarios.Azure_Core_Basic_list = passOnSuccess(
 Scenarios.Azure_Core_Basic_listWithPage = passOnSuccess(
   mockapi.get("/azure/core/basic/page", (req) => {
     const responseBody = {
-      value: [validUser],
```

Comment:
I don't think this one is correct, @tadelesh already fixed the `listWithCustomPage` operation in that PR.

https://github.com/Azure/cadl-ranch/pull/290/files

This operation use the Azure.Core standard `Page`